### ISSUE TITLE
Metastation Update: Progress Report

### ISSUE THREAD

**Issue by The-Albinobigfoot** [Zaers/-tg-station#213]
I'm going to be updating our version of Metastation to be more in line with Metacide's version. I won't be removing anything our own mappers have added (e.g. Xenoarch and the sci shuttle.), and if you see a change you find disagreeable, feel free to speak your mind.

All credit goes to Metacide, and any individuals he specifies in his changelogs.

30-Oct-15
v41A

https://github.com/tgstation/-tg-station/pull/8803

* Added (provisionally) a public autolathe to the RD lobby = REJECTED
* Added (provisionally) a set of chemistry machines to xenobiology (aka secure lab) = ALREADY PRESENT
* Added (provisionally) another independent set of shutters in the HoS office protecting the upper space viewing window = DONE
* Added (provisionally) two drones (RIP) to the derelict station, as well as some metal, glass, fuel, two duffel bags and an airlock painter = REJECTED//Obsolete.
* Added more gear to the vault safe including an anonymous identity and a silenced pistol = REJECTED//Obsolete.
* Added insanity wing to the prison, rearranged it slightly, neatened reeducation room = DONE
* Added derelict auxiliary atmospherics chamber just outside the atmos external airlock, for burn-chamber construction etc. = DONE
* Added a disposal unit to the lower chemistry lab desk and rearranged it to be more usable = DONE
* Added a box of silver IDs to the HoP's room = DONE
* Added two shot-glasses to the maint. bar = DONE
* Added eggs to the Recovery Shuttle kitchen = DONE
* Added fully-functioning biogenerator to the Recovery Shuttle on roundstart = DONE
* Added throwing-stars to the armoury random contraband gun spawner = DONE
* Added teleport computer and missing air alarm to bridge = DONE
* Added collectable HoP hat to the corporate showroom = DONE //Replaced the retractor and surgical drapes with it and a blue folder.
* Added heater to maintenance medbay = ALREADY PRESENT
* Added basketball to maintenance = DONE
* Added suit storage unit to HoS' office = ALREADY PRESENT
* Added hammer and gavel to the courtroom = DONE
* Updated a lot of item paths - banana pies, brig door timers, most pets, and roundstart secure lab slimes = REJECTED//I don't want to fuck with any of that when there doesn't seem to be a need for it.
* Updated syndicate shuttle, station, and CentCom to current Boxstation versions =PARTIALLY DONE/REJECTED (Minor rearrangement in the syndie base, added turrets to the syndie shuttle. Centcom changes put off for now, they're a bit beyond my capabilities currently.)
* Improved NT recovery shuttle (white ship) - it can now dock at mining, has a transit effect, and a slightly altered central room = ALREADY PRESENT/DONE
* Removed non-functional gun charger from the shuttle bridge = ALREADY ABSENT
* Rearranged robotics to give it more open space = DONE
* Rearranged R&D slightly, removed incorrect "mineral storeroom" designation = DONE
* Rearranged lower chemistry machines to be more usable = DONE
* Replaced biohazard containment blast doors around toxins with shutters to stop obscuring open doors = DONE
* Replaced one of the newscasters in the chapel with a notice board = DONE
* Renamed chapel 'mortuary' to funeral parlour = ALREADY PRESENT
* Fixed soulstone shard in chaplain's office being cult/wizard use only - now anyone can use it again = DONE
* Fixed not being able to hide behind plants
* Fixed ERT response room having incorrect map = ALREADY PRESENT
* Fixed overlapping air alarm and fire alarm between showroom and gateway = DONE
* Fixed overlapping no smoking sign and alarm in R&D = DONE
* Fixed overlapping station intercom and map south of gateway = DONE
* Fixed vendors on recovery and emergency shuttle being depowered = DONE
* Fixed incorrect area designations in atmospherics = DONE
* Fixed duplicate science satchels in scientist lockers = DONE
* Fixed broken point redeemer consoles on the labour camp shuttle = ALREADY PRESENT (?)

**Comment by Asd396** [comment]
>public autolathe

Didn't we decide against this in Pizza days? It's piss easy for sci to make one and it makes cargo just a bit less relevant.

**Comment by The-Albinobigfoot** [comment]
Did we? I don't think I was around when that discussion was had. 
I'll remove it for now, you are right about it being trivial to produce.

**Comment by LanCartwright** [comment]
yes. I recall public autolathe has been given the community veto.

**Comment by LanCartwright** [comment]
Take a look at this.

https://tgstation13.org/phpBB/viewtopic.php?f=11&t=5090

**Comment by The-Albinobigfoot** [comment]
/tg/'s lighting system is a bit different than ours, there are procs and vars they've added that just aren't present in ours. Procs and vars essential to the starlight system.
